activities/FractionBounce.activity/index.html

```diff
@@ -118,6 +118,8 @@
 						 v-bind:answer="answer" v-bind:correct-answers="correctAnswers" v-bind:mode="mode" />
 		</div>
 
+		<div id="Score">score:0/10</div>
```

Comment:
The string should be localized.

Reply:
Sure I will try to localize score. I have learned a little bit about it as part of learning how to create our own activity in vanillajs and vuejs.